pkg/phlaredb/block_querier_test.go

```diff
@@ -1386,3 +1388,21 @@ func testSelectMergeByStacktracesRace(t testing.TB, times int) {
 	require.NoError(t, g.Wait())
 	require.NoError(t, querier.Close())
 }
+
+func TestBlockMeta_loadsMetasIndividually(t *testing.T) {
+	path := testDataPath
+	bucket, err := filesystem.NewBucket(path)
+	require.NoError(t, err)
+
+	ctx := context.Background()
+	blockQuerier := NewBlockQuerier(ctx, bucket)
+	metas, err := blockQuerier.BlockMetas(ctx)
+	require.NoError(t, err)
```

Comment:
I'd add something like this to ensure there's actually data to test.
```suggestion
	require.NoError(t, err)
	require.NotEmpty(t, metas)
```

Reply:
good one as well, fixed